installer.tpl.sh

```diff
@@ -267,7 +274,29 @@ install_yum_repo ()
     exit 1
   fi
 
-  cat <<EOF > /etc/yum.repos.d/tarantool_${ver_repo}.repo
+  # For 3 series we use static builds and we can use one repository
+  # for all deb-based and rpm-based systems
+  if [ "${ver}" = 3 ]; then
```

Comment:
All comments above are applied to this as well.

Reply:
fixed